tests/cli/test_ethpm.py

```diff
@@ -53,3 +64,29 @@ def test_ethpm_uninstall(tmp_path, test_assets_dir):
     assert check_dir_trees_equal(
         ethpm_dir, test_assets_dir / "wallet" / "ipfs_uri" / "_ethpm_packages"
     )
+
+
+def test_ethpm_uninstall_nonexistent_package(tmp_path):
+    ethpm_dir = tmp_path / "_ethpm_packages"
```

Comment:
I'm not sure if you're doing this, but to prevent any accidental mucking up of your local working directory I recommend an `autouse=True` fixture in your soot conftest like this:

```python
@pytest.fixture(autouse=True)
def _clean_current_working_directory(monkeypatch):
    with tempfile.tempdir() as temp_dir:
        monkeypatch.chdir(temp_dir)
        yield
```